examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
```

Comment:
No, you can set this to whatever value you want. The only problem is that with too few iterations, PFASST can become unstable over the course of the simulation.